cmd/cmd.go

```diff
@@ -122,11 +128,29 @@ kubectl neat get -- svc -n default myservice --output json`,
 		if err != nil {
 			return err
 		}
-		cmd.Println(string(out))
+		err = printOutput(cmd, out)
+		if err != nil {
+			return err
+		}
 		return nil
 	},
 }
 
+func printOutput(cmd *cobra.Command, out []byte) error {
+	if *color {
+		outreader := bytes.NewReader(out)
+		highlightedOutput, err := highlight.Highlight(outreader)
+		if err != nil {
+			return err
+		}
+		cmd.Println(highlightedOutput)
```

Comment:
please rebase and use `Print` based on https://github.com/itaysk/kubectl-neat/pull/57